posix/stat.c

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/corelibc/sys/stat.h>
+#include <openenclave/internal/posix/device.h>
+#include <openenclave/internal/posix/raise.h>
+#include <openenclave/internal/trace.h>
+#include "mount.h"
+
+int oe_stat(const char* pathname, struct oe_stat* buf)
+{
+    int ret = -1;
+    oe_device_t* fs = NULL;
+    char filepath[OE_PATH_MAX] = {0};
```

Comment:
For my understanding, is zero-initializing the input to `oe_mount_resolve` necessary? We don't do it consistently (see stdio.c and unistd.c) and I just want to confirm it doesn't matter. (i.e. we don't rely on this as a fallback for null termination)

Reply:
Fixed. I though I removed all those but missed a couple. I don't like frivolous initializations.